src/bot/handlers/streamHandler.js

```diff
@@ -0,0 +1,43 @@
+module.exports = class StreamHandler {
+    constructor(streamConfig) {
+        this.streamConfig = streamConfig;
+    }
+
+    async sendPrivateMessage(to, subject, text) {
+        this.streamConfig.wrap.composeMessage({
+            to: to,
+            subject: subject,
+            text: text,
```

Comment:
```suggestion
            text,
```

Kannst du auch bei den beiden Parametern darüber machen